tidb-server/main.go

```diff
@@ -322,7 +323,12 @@ func setGlobalVars() {
 	if cache.PlanCacheEnabled {
 		cache.PlanCacheCapacity = cfg.PlanCache.Capacity
 		cache.PlanCacheShards = cfg.PlanCache.Shards
-		cache.GlobalPlanCache = cache.NewShardedLRUCache(cache.PlanCacheCapacity, cache.PlanCacheShards)
+		cache.GlobalPlanCache = kvcache.NewShardedLRUCache(cache.PlanCacheCapacity, cache.PlanCacheShards)
+	}
+
+	cache.PreparedPlanCacheEnabled = cfg.PreparedPlanCache.Enabled
```

Comment:
`cache.PreparedPlanCacheEnabled` should be initialized when `tidb-server` is started, not here.

Reply:
Since Prepared Plan Cache will be initialized at CreateSession(),  I think that setGlobalVars() can  a right place. Would you suggest a better place?

Reply:
never mind, sorry for this wrong comment